openadapt/evaluators/data_models.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+from typing import Any, Optional
+
+from pydantic import BaseModel
+
+
```

Comment:
Adding `pydantic` to enforce strong type and validation for `action` and `windows`. This helps ensure stability of the evaluator ( and potentially future tuning operation)